### PR TITLE
[WebUI][test][feature] Use HAPI2 plugins as test targets.

### PR DESCRIPTION
**Pull request by kz0817**
This patch is one of preparations for addressing #1938, which
will remove legacy built-in Arms for Zabbix and nagios.
So the tests using them should be fixed to use those of HAPI2
version.

### CODE REVIEW

File: client/test/feature/feature_test_utils.js [hunk 1]
```diff
@@ -66,6 +66,8 @@ function registerMonitoringServer(test, params) {
     });
 
   // emulate actual inputs
+  // TODO: These can be use only for Zabbix (HAPI2). We should support
```
Comment (cosmo0920) [inline client/test/feature/feature_test_utils.js]: このプルリクエストはこのままで問題ないですが、後々 `registerMonitoringServer` から `registerMonitoringServerHAP2Zabbix` とかにリネームかこの関数にoptionを追加してNagiosNDOUtilsやCeilometerもサポートしたいですね！

File: client/test/feature/register_server_test.js [hunk 2]
```diff
@@ -117,23 +127,18 @@ casper.test.begin('Register/Unregister server test', function(test) {
...
       test.assertExists("div.ui-dialog-buttonset > button");
     });
-  // check DOMNodeInserted event
+  // Wait for the response and the addition of the registered server.
   casper.waitFor(function() {
     return this.evaluate(function() {
-      return $(document).on("DOMNodeInserted",
-                            "table tr",
-                            function() {return true;});
+      return $("tr").length >= 2;
```
Comment (cosmo0920) [inline client/test/feature/register_server_test.js]: :+1:

File: client/test/feature/register_server_test.js [hunk 3]
```diff
@@ -232,20 +229,15 @@ casper.test.begin('Register/Unregister server test', function(test) {
     });
   casper.waitFor(function() {
     return this.evaluate(function() {
-      return $(document).on("DOMNodeRemoved",
-                            "table tr",
-                            function() {return true;});
+      return $("tr").length == 1;
```
Comment (cosmo0920) [inline client/test/feature/register_server_test.js]: Sorry for writing ugly code.... :cold_sweat:

**Comment by cosmo0920** [comment]
Looks very nice for me!
Any concerns?
I'll merge this PR tomorrow if no objections.

**Comment by ashie** [comment]
:+1:

**Pull request merged**